app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -50,25 +52,42 @@ class RawSubmissionTable extends React.Component {
       })
       .then(res => {
         this.props.resetSelection();
+
+        let inactive_groups_count = 0;
+        res.groupings.forEach(group => {
+          if (group.members.length && group.members.every(member => member[1])) {
+            group.inactive = true;
+            inactive_groups_count++;
+          } else {
+            group.inactive = false;
+          }
+        });
+
+        this.toggleShowInactiveGroups(null);
+
         const markingStates = getMarkingStates(res.groupings);
+
         this.setState({
           groupings: res.groupings,
           sections: res.sections,
           loading: false,
           marking_states: markingStates,
+          inactiveGroupsCount: inactive_groups_count,
         });
       });
   };
 
   onFilteredChange = (filtered, column) => {
     const summaryTable = this.checkboxTable.getWrappedInstance();
-    if (column.id != "marking_state") {
-      const markingStates = getMarkingStates(summaryTable.state.sortedData);
-      this.setState({marking_states: markingStates});
-    } else {
+    if (column.id == "marking_state") {
```

Comment:
Don't change this, it's not related to the changes you're making

Reply:
Whoops, this was from a previous attempt to implement the functionality, will change.